src/Copy/Text.elm

```diff
@@ -432,40 +432,40 @@ t key =
             "People you can talk to"
 
         GetHelpSection1Title ->
-            "Join the SEA survivors’ forum"
+            "Support from survivors"
 
-        GetHelpSection1Quote1 ->
+        GetHelpSection1Quote ->
             "Meeting other people who’ve been through what I’ve been through is such a massive help. It’s amazing to know people who can really understand what I went through."
 
         GetHelpSection1Description ->
-            "To share your experience with others in our supportive community of economic abuse survivors, sign up to SEA’s Survivor Forum."
+            "Share your experience with others in our supportive community of economic abuse survivors."
 
-        GetHelpSection1Link ->
-            "[cCc] Join SEA Survivor Forum"
+        GetHelpSection1CallToAction ->
+            "Join SEA's survivor forum"
 
         GetHelpSection2Title ->
-            "The Financial Support Line for Victims of Domestic Abuse"
+            "The Financial Support Line"
```

Comment:
We should decide between title case and sentence case for headings to help preserve parity of the options, and general consistency.

Reply:
Yes. Add to your card? It might be time to shard off the copy card from your list and add this as a line item?